inst/qml/common/GlmInputComponent.qml

```diff
@@ -67,7 +59,7 @@ Group {
 		AssignedVariablesList
 		{
 			name:              "weights"
-			title:             updateWeightsTitle()
+			title:             family.currentText === "Binomial" ? qsTr("Total Number of Trials") : qsTr("Weights")
```

Comment:
```suggestion
			title:             family.currentValue === "binomial" ? qsTr("Total Number of Trials") : qsTr("Weights")
```
I think if you check the `text` instead of `value`, it would only be guaranteed to work only in english, as the `text` may get translated...

Reply:
Also tested it on my local machine. The unit tests passed and the interface looks fine.